### DISCUSSION TITLE
Newbie question: How can I move or load or whatever the public/temp folder to /tmp?

### DISCUSSION THREAD

**Discussion by Crehet** [Difegue/LANraragi#813] [Q&A] [answered]
I just want my ssd to not perform reads and writes while am slurping my candies.
im using archlinux

**Comment by Difegue** [comment, marked as answer]
See https://sugoi.gitbook.io/lanraragi/v/dev/extending-lanraragi/architecture#specific-environment-variables-you-can-apply-to-change-lrr-behavior :  

> LRR_TEMP_DIRECTORY - Temporary directory override. If this variable is set to a path, the temporary folder will be there instead of /public/temp.

**Reply by Crehet** [reply]
is it normal to still see reads but no writes from disk? i was expecting to not see reads and writes at all

**Reply by Difegue** [reply]
Well, it's still going to serve the main server files (html/js) from /public, maybe that's what you're seeing? You can probably profile it if you want to know what exactly it's reading

**Reply by Crehet** [reply]
>Well, it's still going to serve the main server files (html/js) from /public, maybe that's what you're seeing?

nah, i doubt it. because the reads were rather high in MiBs

>You can probably profile it if you want to know what exactly it's reading

i don't know how to profile this or that to know what it is reading. but well, overall this is plenty already
